sdk/python/foundation-models/system/inference/text-to-image/aacs-scoring-files/model_download_component/download_model_and_register.yml

```diff
@@ -0,0 +1,35 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+
+name: download_and_register_model
+version: 0.0.0
+type: command
+
+is_deterministic: True
+
+display_name: Download and register model to ws
+description: Component downloads and registers model to workspace for deployement purpose
+inputs:
+  registry_name: 
+    type: string
+  subscription_id:
+    type: string
+  resource_group: 
+    type: string
+  workspace_name: 
+    type: string
+  model_name:
+    type: string
+outputs:
+  local_model:
+    type: uri_folder
+environment: azureml://registries/azureml/environments/model-management/versions/17
```

Comment:
any suggestions on this?

Reply:
Can you check the ones used in import model pipeline?

Reply:
download model component in model converter pipeline has the same env. Better to continue with this env

Reply:
I was under the impression that we are using download_model and register model components directly.
The download_model component can download model from registry.

Reply:
That's what I meant with import mode pipeline above. 
@vadthyavath  - Please take a look at 2.3 of https://github.com/Azure/azureml-examples/blob/main/sdk/python/foundation-models/system/import/import_mmdetection_model_into_registry.ipynb